Readme.md

```diff
@@ -84,44 +84,63 @@ the specifics for *pino-multi-stream*:
 
 [pinoapi]: https://github.com/pinojs/pino#api
 
-<a id="caveats"></a>
-## Caveats
+### pinoms.multistream(streams)
+
+Manually create a single `multistream` as used internally by the
+wrapper:
+
+```js
+var fs = require('fs')
+var pino = require('pino')
+var multistream = require('pino-multi-stream').multistream
+var streams = [
+  {stream: fs.createWriteStream('/tmp/info.stream.out')},
```

Comment:
From Bunyan's documentation:

> The level to which logging to this stream is enabled. If not specified it defaults to "info". If specified this can be one of the level strings ("trace", "debug", ...) or constants (`bunyan.TRACE`, `bunyan.DEBUG`, ...). This serves as a severity threshold for that stream so logs of greater severity will also pass through (i.e. If level="warn", error and fatal will also pass through this stream).

This package does the same thing. The idea is that you can have different destinations for different usages. For example, your normal production may be at `error` or `info` level, but you _also_ want `debug` level shipped to a server for developers